### ISSUE TITLE
Terraria-Server Install can't enter Steam-Guard-Code

### ISSUE THREAD

**Issue by DaMaxx** [dgibbs64/linuxgsm#892]
Hello Guys,

I tried installing a terraria-server on my Debian machine, but the installation fails at the point where I'm supposed to enter my Steam-Guard-Code (I tried the mobile authenticator and the E-Mail one).

The installation asks automatically for my key, and I'm able to enter the code, but hitting return or enter doesn't seem to have an effect.

`Logging in user 'XXXXXXX' to Steam Public...
Enter the current code from your Steam Guard Mobile Authenticator app
Two-factor code:XXXXX`

The script doesn't continue after that. I guess it's a problem with the script itself since the script for the 7 Days to Die server works fine.

Systeminformation:
Debian GNU/Linux 8
Linux  XXXX.org 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt25-2 (2016-04-08) x86_64 GNU/Linux

(Removed personal information)

**Comment by UltimateByte** [comment]
We didn't figure out how to fix the steam guard yet.
https://github.com/dgibbs64/linuxgsm/issues/579
